src/tasks/minions/bso/tinkeringWorkshopActivity.ts

```diff
@@ -0,0 +1,79 @@
+import { randArrItem, randInt, roll } from 'e';
+import { Bank } from 'oldschooljs';
+import { ItemBank } from 'oldschooljs/dist/meta/types';
+
+import { userHasFlappy } from '../../../lib/invention/inventions';
+import { MaterialBank } from '../../../lib/invention/MaterialBank';
+import { incrementMinigameScore } from '../../../lib/settings/settings';
+import { ClueTable } from '../../../lib/simulation/sharedTables';
+import { SkillsEnum } from '../../../lib/skilling/types';
+import { TinkeringWorkshopOptions } from '../../../lib/types/minions';
+import { handleTripFinish } from '../../../lib/util/handleTripFinish';
+import resolveItems from '../../../lib/util/resolveItems';
+import { userStatsUpdate } from '../../../mahoji/mahojiSettings';
+
+const inventorOutfit = resolveItems([
+	"Inventors' helmet",
+	"Inventors' torso",
+	"Inventors' legs",
+	"Inventors' gloves",
+	"Inventors' boots",
+	"Inventors' backpack"
+]);
+function tinkerLoot(user: MUser, quantity: number) {
+	const loot = new Bank();
+	let effectiveBank = user.bank.clone();
+	for (let i = 0; i < quantity; i++) {
+		const outfitPieceNotOwned = randArrItem(inventorOutfit.filter(p => !effectiveBank.has(p)));
+		if (roll(6)) {
+			loot.add(ClueTable.roll());
+		}
+		if (roll(120)) loot.add('Materials bag');
+		if (outfitPieceNotOwned && roll(16)) {
+			loot.add(outfitPieceNotOwned);
+			effectiveBank.add(outfitPieceNotOwned);
+			continue;
+		}
+	}
+	return loot;
+}
+
+export const twTask: MinionTask = {
+	type: 'TinkeringWorkshop',
+	async run(data: TinkeringWorkshopOptions) {
+		const { channelID, quantity, duration, userID } = data;
+
+		await incrementMinigameScore(userID, 'tinkering_workshop', quantity);
+
+		const user = await mUserFetch(userID);
+		const loot = tinkerLoot(user, quantity);
+
+		const flappyRes = await userHasFlappy({ user, duration });
```

Comment:
Other than clues, what loot would you want to double? The bag + inventors' aren't tradeable, and i dont' see any other loot worth doubling other than clues.

Just a thought